packages/lib-classifier/src/plugins/drawingTools/components/Ellipse/Ellipse.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import DragHandle from '../DragHandle'
+
+const GUIDE_DASH = [4, 4]
+const GUIDE_WIDTH = 1
+
+function Ellipse ({ active, children, mark, onFinish, scale, svg, tool }) {
```

Comment:
`tool` isn't being used here. `svg` should switch to use context.

Reply:
`svg` should be deleted, I think.